### ISSUE TITLE
MultilinePipe Linter doesn't allow text content to end in a space followed by a pipe

### ISSUE THREAD

**Issue by lencioni** [sds/haml-lint#4]
The following HAML is incorrectly reported:

``` haml
%p
  My text content |
  = my_variable
```
